packages/shorebird_cli/lib/src/executables/gradlew.dart

```diff
@@ -75,15 +73,40 @@ class Gradlew {
     }
 
     final executablePath = executableFile.path;
-    final result = await process.run(
+    return process.run(
       executablePath,
-      ['app:tasks', '--all', '--console=auto'],
+      args,
       runInShell: true,
       workingDirectory: p.dirname(executablePath),
       environment: {
         if (!javaHome.isNullOrEmpty) 'JAVA_HOME': javaHome!,
       },
     );
+  }
+
+  /// Returns whether the gradle wrapper exists at [projectPath].
+  bool exists(String projectPath) {
+    final androidRoot = Directory(p.join(projectPath, 'android'));
+    return File(p.join(androidRoot.path, executable)).existsSync();
+  }
+
+  /// Return the version of the gradle wrapper at [projectPath].
```

Comment:
Is `projectPath` the root to the flutter project? Might be better to call it `projectRoot` if so

Reply:
Yeah, `projectPath` was already used through the file, so I kept it like that for the new method.

I agree that `projectRoot` will make more sense, I will change in the new method and refactor in the other places